wiki/Article_Styling_Criteria/en.md

```diff
@@ -86,9 +86,13 @@ It will one day be the successor of the previous osu!client.
 Please, do **not** send the development team any issues you are facing, as things are still subject to change.
 
 Do:
-_osu!_ is a circle clicking simulator with three other game modes. There is drum beating, fruit catching, and key mashing game modes; otherwise known as osu!taiko, osu!catch, and osu!mania respectively.
+osu!lazer is a development build that you can download. It will one day be the successor of the previous osu!client. Please, do **not** send the development team any issues you are facing, as things are still subject to change.
 ```
 
+---
+
+Any Markdown syntax characters that are not being used to style must be escaped, unless it is inside [code](#code).

```

Comment:
L94: Consider Reivising Grammar-wise ``[..] Any Markdown syntax that are not intentionally used to style a specific wording/ phrase/etc. should be escaped, unless it's inside a [code block][...]``

Reply:
you may also want to provide an example of escaping ``[\_yu68]()`` for clarity's sake